@xen-orchestra/backups/_cleanVm.js

```diff
@@ -277,9 +294,11 @@ exports.cleanVm = async function cleanVm(
       if (!vhds.has(parent)) {
         vhds.delete(vhdPath)
 
-        onLog(`the parent ${parent} of the VHD ${vhdPath} is missing`)
+        // `the parent ${parent} of the VHD ${vhdPath} is missing`
+        logWarn('parent VHD is missing', { parent, vhdPath })
```

Comment:
instead of doing

   logWarn('something detected')
   if(remove) { logWarn('really deleting') }
   
 What do you think of
 
    if(remove) { logWarn('really deleting') }
    else {logWarn('something detected') }

Reply:
If the problem is that there are two warnings for a single problem, I believe that logging the actions as info is enough to fix it.